multi-server.js

```diff
@@ -0,0 +1,45 @@
+const express = require('express')
+const http = require('http')
+
+const PORT = parseInt(process.env.PORT || '3000')
+let BACKENDS = []
+
+function initModules (modules) {
+    BACKENDS = modules.map((m) => require(m))
+}
+
+async function initServer (port) {
+    const app = await initApp()
+    app.set('port', port)
+    const server = http.createServer(app)
+    await Promise.all(BACKENDS.map(async (back) => {
+        if (!back.prepareBackServer) return
+        await back.prepareBackServer(server)
+        console.log(`MULTI-SERVER: PATCH SERVER BY ${back.NAME}`)
+    }))
+    return server.listen(port)
+}
+
+async function initApp () {
+    const app = express()
+    await Promise.all(BACKENDS.map(async (back) => {
+        if (!back.prepareBackApp) return
+        const backApp = await back.prepareBackApp()
+        if (!backApp) return
+        app.use(back.URL_PREFIX, backApp)
+        console.log(`MULTI-SERVER: USE APP ${back.NAME} AT URL ${back.URL_PREFIX}`)
+    }))
+    return app
+}
+
+const modules = process.argv.slice(2);
```

Comment:
Am I right by saying that we have to create a `multi-server.js` file in every application we want to include here ?